Sources/StrongContractResponse/All/DataProtocol.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  DataProtocol.swift
+//
+//
+//  Created by Leonardo Dabus on 10/05/24.
+//
+
+import Foundation
+
+#if canImport(Crypto)
+import Crypto
+#endif
+
+#if canImport(CryptoKit)
+import CryptoKit
+#endif
```

Comment:
Add a comment explaining when one can be imported and when the other can't for each.